### ISSUE TITLE
Build steps for ARM binaries.

### ISSUE THREAD

**Issue by NithyeshS** [bfishman/node-webrtc-rpi-binaries#1]
Could you please upload the build steps?

I have cloned https://github.com/bfishman/node-webrtc.git and tried running the `bin/build.js` with the `--arm` flag, but I am unsuccessful. Do I have to set any env variables?

**Comment by vgokul129** [comment]
Yes build procedure please. It will be quite useful for others too.

**Comment by volumio** [comment]
+1

**Comment by bfishman** [comment]
Long story short, I don't have reproducible build steps and do not plan to create them. Sorry.

Honestly the build procedure was unreliable at best. I attempted to make it reproducible but was unable to do so from scratch, meaning there was some unidentified environment variable or system configuration / software existing on my system at the time that was apparently critical to the build. This is not my area of expertise. Eventually I was unable to devote more time to trying to make the build steps reproducible and had to move on, only posting the compiled files here for posterity. They are undoubtedly well behind the master branch now. I may delete this fork.

**Comment by temaivanoff** [comment]
how to use it with node-webrtc?
